pubsub/api/Pubsub.Samples/PublishMessageWithRetrySettingsAsync.cs

```diff
@@ -32,19 +32,20 @@ public async Task PublishMessageWithRetrySettingsAsync(string projectId, string
         var backoffMultiplier = 1.3; // default: 1.0
         var totalTimeout = TimeSpan.FromSeconds(100); // default: 600 seconds
 
-        var publisher = await PublisherClient.CreateAsync(topicName,
-               clientCreationSettings: new PublisherClient.ClientCreationSettings(
-                   publisherServiceApiSettings: new PublisherServiceApiSettings
-                   {
-                       PublishSettings = CallSettings.FromRetry(RetrySettings.FromExponentialBackoff(
+        var publisher = await new PublisherClientBuilder
+        {
+            TopicName = topicName,
+            ApiSettings = new PublisherServiceApiSettings
+            {
+                PublishSettings = CallSettings.FromRetry(RetrySettings.FromExponentialBackoff(
```

Comment:
It might be worth separating out the RetrySettings construction into a separate statement (and variable) beforehand. That can happen later though.

Reply:
Yes, I didn't  want to make many changes here, and there's definetely a mix of settings being created in-line or beforehand across different samples that we should look into. But yes, I don't have the time for that right now. I'm adding a low priority TODO for myself but also @Rishabh-V as FYI, you might be able to look into this at some point.

Reply:
Sure Amanda. Thanks.